go/src/koding/klientctl/ctlcli/ctlcli.go

```diff
@@ -116,9 +116,10 @@ func ExitErrAction(f ExitingErrCommand, log logging.Logger, cmdName string) cli.
 		}
 
 		Close()
-		ExitFunc(exit)
```

Comment:
The way I'm workarounding this design is to register a function that will get executed on exit - ctlcli.OnExit. I'm not sure if making the exit code 1 won't break any behaviour somewhere.